instagram/src/components/CommentSection/Comments.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+
+class Comments extends Component {
+	constructor() {
+		super()
```

Comment:
`super()` calls the constructor of its parent class. This is required when you need to access some variables from the parent class. In here, what variables are you accessing??